redpanda/values_add_tls.yaml

```diff
@@ -76,6 +76,7 @@ config:
 
 certIssuers:
   - name: kafka
+    duration: 8760h
```

Comment:
Do we need to have this in the config? Could we just use the default?

Reply:
We should use the default. I didn't remove the override after testing.